src/run.rs

```diff
@@ -227,6 +232,60 @@ pub(crate) fn run_repl(
         use_color,
     );
 
+    let delta = {
+        let name = "std".to_string();
+        let content = get_standard_library().as_bytes();
+
+        let mut working_set = StateWorkingSet::new(engine_state);
+
+        let start = working_set.next_span_start();
+        working_set.add_file(name.clone(), content);
+        let end = working_set.next_span_start();
+
+        let (_, module, _, _) = parse_module_block(
+            &mut working_set,
+            Span::new(start, end),
+            name.as_bytes(),
+            &[],
+        );
+
+        // TODO: change this when #8505 is merged
+        // NOTE: remove the assert and uncomment the `help`s
+        let prelude = vec![
+            ("assert", "assert"),
+            // ("help", "help"),
+            // ("help commands", "help commands"),
+            // ("help aliases", "help aliases"),
+            // ("help modules", "help modules"),
+            // ("help externs", "help externs"),
+            // ("help operators", "help operators"),
+        ];
+
+        working_set.use_decls(
+            prelude
+                .iter()
+                .map(|(name, search_name)| {
+                    (
+                        name.as_bytes().to_vec(),
+                        module
+                            .decls
+                            .get(&search_name.as_bytes().to_vec())
+                            .unwrap_or_else(|| {
+                                panic!("could not load `{}` from `std`.", search_name)
```

Comment:
We can't panic here, it should probably print an error and continue running. There are `report_error()` and `report_error_new()` functions for printing the error.

Reply:
it does not appear to solve the issue, but still in d0101ddf5 :ok_hand: